test/bert/tokenizer/tBERTTokenizer.m

```diff
@@ -125,6 +125,17 @@ function errorsForDifferentNumberOfSentencePairObservations(test, VocabFiles)
             test.verifyError(@() tok.encode(x1,x2), 'bert:tokenizer:SentencePairNumelMismatch');
         end
         
+        function canReturnWordStartingIndices(test)
+            % The tokenizer splits baz and adds markers;
+            % the other tokens are "original."
+            x = "foo, bar. Baz!";
+            tok = test.Constructor();
+            [~,tokIdx] = tok.encode(x);
```

Comment:
I think we should verify the paired version too. In particular:
```
x1 = "foo bar baz"; x2 = "anything";
[~,tokIdx] = tok.encode(x1,x2);
```
Since `x1` ends with a sub-token, and `[SEP]` is mapped to `0`, I assume the workflow would be to find where `[SEP]` is first, then split up `tokIdx` so that "ba" "##z" "[SEP]" doesn't get interpreted as a single word.